source/documentation/metrics-install-guide/Upgrading_Metrics_Store.html.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Replacing Fluentd with Rsyslog
+---
+
+# Chapter 5: Upgrading Metrics from 4.2
+
+in oVirt 4.3 Fluentd was replaced  by Rsyslog to gather metrics and logs from the oVirt hosts and engine.
+
+Rsyslog is used to collect and parse Engine, VDSM logs and Collectd metrics, add additional metadata and send the data to the remote metrics store.
+
+Note: this does not upgrade the Metrics store but the collectors.
+
+## pre-req
+
+* oVirt Metrics 4.3 with working metrics OpenShift deployment.
+* oVirt host 4.3 which was previously upgraded from 4.2.
+
+
+
+## Procedure
+
+1. Log in to the upgraded Ovirt Host machine using SSH.
+2. Copy `config.yml.example` to create `config.yml`:
```

Comment:
Check `config.yml.example` since config.yml already exists and they may or may not need to update it/
@eslutsky